content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,115 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved**fix readiness probe inconsistency #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings per host #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Enabled** [automatic protocol detection](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection-experimental) by default #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthenticaiton` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** Auto mutual TLS is in Beta. The feature is enabled by default.
+- **Added** SDS is in Stable. SDS is enabled by default for identity provisioning.
+- **Improved** Node agent is merged with Pilot agent. This removes the requirement of pod security policy and improves security posture.
+- **Improved** Merge Citadel certificate provisioning functionality into Pilot.
+- **Improved** Support Kubernetes `first-party-jwt` as a fallback token for CSR authentication in clusters that `third-party-jwt` is not supported.
+- **Improved** Provision key and cert to Prometheus through Istio Agent.
+
```

Comment:
Can you add the following item to the security section?

- **Improved** Support Citadel to provision the control plane certificate.